tests/integration/ossm_installer_int_test.go

```diff
@@ -82,13 +83,102 @@ var _ = When("Migrating Data Science Projects", func() {
 		Eventually(findMigratedNamespaces, timeout, interval).Should(
 			And(
 				HaveLen(3),
-				ContainElements("dsp-01", "dsp-02", "dsp-03")
-		),
+				ContainElements("dsp-01", "dsp-02", "dsp-03"),
+			),
 		)
 	})
 
 })
 
+var _ = When("Checking for CRD", func() {
+	var (
+		ossmInstaller *ossm.OssmInstaller
+	)
+
+	BeforeEach(func() {
+		ossmInstaller = ossm.NewOssmInstaller(&kfconfig.KfConfig{}, envTest.Config)
+	})
+
+	It("should successfully check existing CRD", func() {
+		// given example CRD installed into env from /ossm/test/crd/
+		crdGroup := "ossm.plugins.kubeflow.org"
+		crdVersion := "test-version"
+		crdResource := "test-resources"
+
+		// when
+		err := ossmInstaller.CheckForCRD(crdGroup, crdVersion, crdResource)
+
+		// then
+		Expect(err).To(BeNil())
+	})
+
+	It("should fail to check non-existing CRD", func() {
+		// given
+		crdGroup := "non-existing-group"
+		crdVersion := "non-existing-version"
+		crdResource := "non-existing-resource"
+
+		// when
+		err := ossmInstaller.CheckForCRD(crdGroup, crdVersion, crdResource)
+
+		// then
+		Expect(err).To(HaveOccurred())
+	})
+})
+
+var _ = When("Checking for SMCP", func() {
```

Comment:
```suggestion
var _ = When("Ensuring environment is set up correctly", func() {
```

Was wondering if naming by feature/responsibility vs implementation detail is better suited. WDYT?